src/subscription_manager/cache.py

```diff
@@ -20,6 +20,7 @@
 this with the current state, and perform an update on the server if
 necessary.
 """
+import copy
```

Comment:
This import is breaking the flake8 stylish tests, IMO it should be removed.

Reply:
It didn't break nosetests for me.  Are you running the tests differently (that way I will know if I need to run additional tests).  But I removed.